src/Testcontainers.DockerCompose/DockerCompose.cs

```diff
@@ -0,0 +1,64 @@
+﻿namespace Testcontainers.DockerCompose;
+
+/// <inheritdoc cref="DockerContainer" />
+internal abstract class DockerCompose : DockerContainer
+{
+    private static readonly IList<string> AppLineArgs = new[] { "docker", "compose"};
+    
+    private static readonly IList<string> StartCommandLineArgs = new[] { "up", "-d" };
+    private static readonly IList<string> StopCommandLineArgs = new[] { "down" };
+
+    private static readonly IList<string> RemoveImagesArgs = new[] { "--rmi" };
+    private static readonly IList<string> FilesArgs = new[] { "-f" };
+
+    /// <summary>   
+    /// Initializes a new instance of the <see cref="DockerCompose" /> class.
+    /// </summary>
+    protected DockerCompose(IContainerConfiguration configuration, ILogger logger) : base(configuration, logger)
+    {
+    }
+
+    /// <summary>
+    /// Gets the runtime configuration.
+    /// </summary>
+    public DockerComposeConfiguration RuntimeConfiguration => _configuration as DockerComposeConfiguration;
```

Comment:
The additional runtime configuration was needed to be able to work with it after the container is started. If we remove `ConfigureDockerComposeAsync`, it will probably not be needed.